src/olympia/shelves/tests/test_views.py

```diff
@@ -231,7 +231,9 @@ def test_adzerk_returns_none_sponsored(self):
                     'impression': '',
                     'click': ''},
             }
-            data = self.perform_search()
+            with mock.patch('django_statsd.clients.statsd.incr') as incr_mock:
+                data = self.perform_search()
+            incr_mock.assert_any_call('services.adzerk.elasticsearch_miss', 3)
```

Comment:
Do we need to assert about the log call as well? I realize we probably don't assert about log calls much, but this is specifically being added to address an issue from the RRA, as opposed to just support possible debugging. We wouldn't want it accidentally removed.

Reply:
https://github.com/mozilla/addons-server/pull/15928/commits/18eccf2fce952c87ff04d63ca84c70972fe02a1a